File: R/example_usage.R
# example_usage.R
# Example usage of the calculate_eigen function

# Define a sample matrix
matrix <- matrix(c(4, 1, 2, 3), nrow = 2, byrow = TRUE)

# Calculate eigenvalues and eigenvectors
eigen_results <- calculate_eigen(matrix)

# Display the results
cat("Eigenvalues:\n")
print(eigen_results$eigenvalues)
cat("\nEigenvectors:\n")
print(eigen_results$eigenvectors)

File: Python/example_usage.py
# example_usage.py
# Example usage of the calculate_eigen function

from eigen_calculation import calculate_eigen

# Define a matrix (for example, a 3x3 matrix)
matrix = [
    [4, -2, 1],
    [1, 1, -1],
    [3, -1, 2]
]

# Calculate eigenvalues and eigenvectors
eigenvalues, eigenvectors = calculate_eigen(matrix)

# Print the results
print("Eigenvalues:")
print(eigenvalues)

print("\nEigenvectors:")
print(eigenvectors)
